src/libraries/System.Private.CoreLib/src/System/Threading/PreAllocatedOverlapped.Portable.Core.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Threading
+{
+    /// <summary>
+    /// Represents pre-allocated state for native overlapped I/O operations.
+    /// </summary>
+    /// <seealso cref="ThreadPoolBoundHandle.AllocateNativeOverlapped(PreAllocatedOverlapped)"/>
+    public sealed partial class PreAllocatedOverlapped : IDisposable, IDeferredDisposable
+    {
+        internal readonly ThreadPoolBoundHandleOverlapped? _overlapped_portable_core;
```

Comment:
coding style is camel case `_overlappedPortableCore` 

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md